docs/src/Components.jsx

```diff
@@ -90,6 +90,22 @@ var Components = React.createClass({
           </ul>
         </article>
 
+        <article id="lozenge">
+          <h3>Lozenge</h3>
+          <p>A small pill shaped item for displaying information</p>
+          <CustomComponent codeText={fs.readFileSync(__dirname + '/../examples/Lozenge.jsx', 'utf8')} />
+          <h4>Attributes</h4>
+          <ul>
+            <li><code>tip</code> String - Tooltip text to display</li>
+            <li><code>purpose</code> String - Specifies purpose and adds relevant style</li>
+            <li><code>info</code> Boolean - Adds info style</li>
```

Comment:
In all other components we pass through `purpose='danger'` etc maybe we should keep this consistent